apps/sms/services/README.md

```diff
@@ -0,0 +1,114 @@
+# The layers in the Messages application
+
+## Terms explanation
+
+We'll define different layers whose interfaces managed by the Bridge (aka [threads.js](https://github.com/gaia-components/threads/)):
+
+* Controllers are really tied to a specific view. They'll encapsulate
```

Comment:
Basically "controller" it's a part of ```views/inbox/inbox.js```, ```views/conversation/conversation.js``` that directly isn't related to updating DOM?